src/CounterInterceptor/MyWorkflow.workflow.cs

```diff
@@ -0,0 +1,42 @@
+namespace TemporalioSamples.CounterInterceptor;
+
+using Temporalio.Workflows;
+
+[Workflow]
+public class MyWorkflow
+{
+    private bool exit; // Automatically defaults to false
+
+    [WorkflowRun]
+    public async Task<string> RunAsync()
+    {
+        // Wait for greeting info
+        await Workflow.WaitConditionAsync(() => Name != null && Title != null);
```

Comment:
These properties can never be null, so this statement has no value, it will always pass immediately

Reply:
Bump, this was never addressed

Reply:
Bump again, this is still not addressed

Reply:
Fixed.